### ISSUE TITLE
[Translatable][Symfony] Undefined constant Doctrine\DBAL\Types\Type::BIGINT

### ISSUE THREAD

**Issue by Cybercraft** [doctrine-extensions/DoctrineExtensions#2275]
**Describe the bug**
The way to get the type of foreignKey into the ORM.php below the Translatabe\Mapping\Event\Adapter namespace is no more compatible with Doctrine DBAL Types constant.

The incompatibilty throw an exception when you try to get a set of tranlsatable entities from the database (in this case)

In details :

```php
private function foreignKey($key, $className)
    {
        $em = $this->getObjectManager();
        $meta = $em->getClassMetadata($className);
        $type = Type::getType($meta->getTypeOfField('foreignKey'));
        switch ($type->getName()) {
        // These constant throw the exception. use Types instead of Type
        case Type::BIGINT:
        case Type::INTEGER:
        case Type::SMALLINT:
            return intval($key);
        default:
            return (string) $key;
        }
    }
```

**To Reproduce**
1. Setup a Symfony 5.3.9 in PHP 8.0
2. Create new Translatable entity and her repository
3. Generate some entities through a Fixtures for example.
4. Create a controller and fetch entities from Database with ( <Entity>Repository->findAll() )

**Expected behavior**
I expect to get all items from the database.

**Additional context**
I think you can use the final class Types instead of abstract Type for getting constants. Doctrine use Types instead of Type.

**Comment by techws-dev** [comment]
the dependency on **gedmo/doctrine-extensions** should be updated to 3.2.0, this has been resolved 
[here](https://github.com/doctrine-extensions/DoctrineExtensions/commit/32c5f3941dfe60a99dfe27cf528c67d42106db59)

**Comment by phansys** [comment]
Thank you for the tip @techws-dev!
@Cybercraft, indeed, this issue was fixed in #2250. It is released under [3.2.0](https://github.com/doctrine-extensions/DoctrineExtensions/releases/tag/v3.2.0).

I'm closing this issue. Please, feel free to open again is something is missing.
